### PR TITLE
meta server addr configurable

### PR DESCRIPTION
**Pull request by TianTianBigWang**
Signed-off-by: TianTianBigWang <zilong.wang@daocloud.io>

**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by gy95** [comment]
also need to do some work that we can use `edgecore --defaultconfig > edgecore.yaml` cmd to generate this default config.

**Comment by gy95** [comment]
oh, sorry, ignore the previous comment, variable `Httpaddr` is also used in the test file

File: pkg/apis/componentconfig/edgecore/v1alpha1/default.go [hunk 1]
```diff
@@ -135,6 +135,7 @@ func NewDefaultEdgeCoreConfig() *EdgeCoreConfig {
 				MetaServer: &MetaServer{
 					Enable: false,
 					Debug:  false,
+					Server: "127.0.0.1:10550",
```
Comment (zc2638) [inline pkg/apis/componentconfig/edgecore/v1alpha1/default.go:138]: You can set it as a constant and reference it elsewhere.

File: tests/integration/metaserver/access_test.go [hunk 2]
```diff
@@ -10,12 +10,12 @@ import (
...
 	"k8s.io/klog/v2"
 
-	"github.com/kubeedge/kubeedge/edge/pkg/metamanager/metaserver"
+	"github.com/kubeedge/kubeedge/common/constants"
 )
 
 func TestGet(t *testing.T) {
 	kubeclient, err := kubernetes.NewForConfig(&rest.Config{
-		Host: "http://" + metaserver.Httpaddr,
+		Host: constants.DefaultMetaServerAddr,
```
Comment (zc2638) [inline tests/integration/metaserver/access_test.go:18]: Need to add protocol?

Comment (TianTianBigWang) [reply tests/integration/metaserver/access_test.go:18]: no

Comment (TianTianBigWang) [reply tests/integration/metaserver/access_test.go:18]: may be yes

File: pkg/apis/componentconfig/edgecore/v1alpha1/types.go [hunk 3]
```diff
@@ -383,8 +383,9 @@ type MetaManager struct {
 }
 
 type MetaServer struct {
-	Enable bool `json:"enable"`
-	Debug  bool `json:"debug"`
+	Enable bool   `json:"enable"`
+	Debug  bool   `json:"debug"`
+	Server string `json:"server"`
```
Comment (fisherxu) [inline pkg/apis/componentconfig/edgecore/v1alpha1/types.go:388]: Do  we  need to spilit the server and port in config?

Comment (zc2638) [reply pkg/apis/componentconfig/edgecore/v1alpha1/types.go:388]: No other place needs a port, it doesn’t have to be alone.

**Review by fisherxu — COMMENTED** [review]
/approve

**Pull request merged**